packages/now-cli/test/integration.js

```diff
@@ -753,47 +670,19 @@ test('set platform version using `-V` to `2`', async t => {
   console.log(stdout);
   console.log(exitCode);
 
-  const output = `Received:\n"${stderr}"\n"${stdout}"`;
-
-  // Ensure the exit code is right
-  t.is(exitCode, 0, output);
-
-  // Test if the output is really a URL
-  const { href, host } = new URL(stdout);
-  t.is(host.split('-')[0], session, output);
-
-  if (host) {
-    context.deployment = host;
-  }
-
-  // Send a test request to the deployment
-  const response = await fetch(href);
-  const contentType = response.headers.get('content-type');
-
-  t.is(contentType, 'text/html; charset=utf-8');
-});
-
-test('output logs of a 2.0 deployment', async t => {
-  const { stderr, stdout, exitCode } = await execa(
-    binaryPath,
-    ['logs', context.deployment, ...defaultArgs],
-    {
-      reject: false,
-    }
+  t.true(
+    stderr.includes(`Fetched deployment "${deployment}"`),
+    formatOutput({ stderr, stdout })
   );
-
-  console.log(stderr);
-  console.log(stdout);
-  console.log(exitCode);
-
-  t.true(stderr.includes(`Fetched deployment "${context.deployment}"`));
   t.is(exitCode, 0);
 });
 
 test('output logs of a 2.0 deployment without annotate', async t => {
+  const deployment = await createDeploymentWithFixture();
```

Comment:
This might make our tests a little slower since its creating a deployment for each test. Previously, this `logs` test was reading the logs of the previous deployment.

Reply:
@styfle I am aware, but ideally the deployment should be deduped, so it's not that much overhead. And I want every test to work on it's own, so that we can remove the `context` stuff and use `test.only` without running into any issues because of that.